Basic/APIDiorama/Assets/Common/Prefabs.meta

```diff
@@ -0,0 +1,8 @@
+fileFormatVersion: 2
```

Comment:
I'm thinking that the level that is part of every scene can be made into its own prefab, since I don't think anything changes in terms of colliders/physics/tags.
We could also do this for the UICamera, Post-process volume, as well as the scene camera.

Reply:
I agree, done!